src/app/delegation_backend/scripts/build.sh

```diff
@@ -22,6 +22,14 @@ else
 fi
 cp "$ref_signer"/*.h "$OUT/headers"
 
+docker_toolchain_build() {
```

Comment:
Can you move these docker build commands to `./scripts/release-docker.sh` and symlink the dockerfile into `dockerfiles/` this way all of the containers are built + stored in the same places